packages/editor/src/pages/Editor/store/solutions/sagas.ts

```diff
@@ -41,7 +45,7 @@ function* onSolutionOpenOrFileEditSaga(
           selectors.solutions.getFile,
           action.payload.fileId,
         );
-        if (file.language === 'typescript') {
+        if (file.language === 'typescript' || file.language === 'python') {
```

Comment:
nit: *some SCRIPT_LANGUAGES const string[] somewhere*
`SCRIPT_LANGUAGES.includes(file.language)`

Reply:
OK, fair point.  But let me defer this to another time.  I don't think it's worth doing a full rebuild just for this, and I'd rather get this change into alpha sooner.